actors/market/src/types.rs

```diff
@@ -95,11 +99,31 @@ pub struct ActivateDealsParams {
     pub sector_expiry: ChainEpoch,
 }
 
+#[derive(Serialize_tuple, Deserialize_tuple, Clone)]
+pub struct VerifiedDealInfo {
+    pub client: ActorID,
+    pub allocation_id: AllocationID,
+    pub data: Cid,
+    pub size: PaddedPieceSize,
+}
+
+impl Default for VerifiedDealInfo {
+    fn default() -> VerifiedDealInfo {
+        VerifiedDealInfo {
+            size: PaddedPieceSize(0),
+            client: 0,
+            allocation_id: 0,
+            data: Default::default(),
+        }
+    }
```

Comment:
I'm a little worried this could lead to an error, since it could never be a valid info. I can see its use in testing, but if it's not too hard to fill in with other values, I'd rather remove this default impl.